README.md

```diff
@@ -469,6 +470,10 @@ Follow the [FAQ](docs/FAQ.md) page.
 
 Follow the [Development](docs/Development.md) guide. It has all the information on how to get started.
 
+## Architectural designs
+
+Follow the [Architectural designs](docs/design/ArchitecturalDesigns.md) document that describes and documents XCRemoteCache designs and implementation details.
```

Comment:
👍 Minor - the graphs are hard to read for dark mode.

Reply:
thanks for spotting that! Fixed it.